### ISSUE TITLE
Dashboard displays errors

### ISSUE THREAD

**Issue by ejstembler** [WordPress/health-check#359]
### Feature request/bug description

The dashboard displays the following errors:

```console
Notice: Trying to get property of non-object in /var/www/wp-content/plugins/health-check/includes/class-health-check-dashboard-widget.php on line 35

Notice: Trying to get property of non-object in /var/www/wp-content/plugins/health-check/includes/class-health-check-dashboard-widget.php on line 50
```

### Screenshots

![site-health-status-errors](https://user-images.githubusercontent.com/45985338/63907483-a1c33a80-c9e9-11e9-9906-5ae7927d8854.png)

**Comment by ejstembler** [comment]
Incidentially, I turned off debugging in `wp-config.php` and the messages went away:

```php
define('WP_DEBUG', false);
```

**Comment by Clorith** [comment]
Hiya,

This is due to an oversight, where an array was used as a fallback, instead of an object. This has been corrected in 229cf91b2db72ded9a9fd51c7264d4a0928f219f and will be included in the next update (v1.4.1)
